torch/_inductor/exc.py

```diff
@@ -1,28 +1,28 @@
-# mypy: allow-untyped-defs
 from __future__ import annotations
 
 import os
 import tempfile
 import textwrap
 from functools import lru_cache
+from typing import Any
 
 
 if os.environ.get("TORCHINDUCTOR_WRITE_MISSING_OPS") == "1":
 
     @lru_cache(None)
-    def _record_missing_op(target):
+    def _record_missing_op(target: Any) -> None:
```

Comment:
Technically, it needs to be something is convertable to str, but surprisingly SupportsStr is not one of the default protocls in TypingExtensions (it would be trivial to add here though).